tests/unit/test_predictor.py

```diff
@@ -132,7 +133,7 @@ def json_sagemaker_session():
     response_body.close = Mock("close", return_value=None)
     ims.sagemaker_runtime_client.invoke_endpoint = Mock(
         name="invoke_endpoint",
-        return_value={"Body": response_body, "ContentType": DEFAULT_CONTENT_TYPE},
+        return_value={"Body": response_body, "ContentType": "application/json"},
```

Comment:
This fix is out of scope for this PR, but since it's small I decided to keep it in.